### PR TITLE
Fix up authority part

### LINKED ISSUE #8199: Url.GetLeftPart wrong cells order in Remarks
`<none>` value should be moved left to Authority column for `mailto:user@contoso.com?subject=uri` and `news:123456@contoso.com`

https://github.com/dotnet/dotnet-api-docs/blob/12bdd60af5cc042c37d9a486136619434656dfc3/xml/System/Uri.xml#L2415

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #8199.

### CODE REVIEW

**Comment by CarnaViire** [comment]
cc @MihaZupan

**Comment by gewarren** [comment]
@MihaZupan Could you review this?

File: xml/System/Uri.xml [hunk 1]
```diff
@@ -2394,27 +2394,25 @@ If you used an escaped string to construct this instance (for example, `"http://
...
-
- The following examples show a URI and the results of calling <xref:System.Uri.GetLeftPart%2A> with <xref:System.UriPartial.Scheme>, <xref:System.UriPartial.Authority>, <xref:System.UriPartial.Path>, or <xref:System.UriPartial.Query>.
+The following examples show a URI and the results of calling <xref:System.Uri.GetLeftPart%2A> with <xref:System.UriPartial.Scheme>, <xref:System.UriPartial.Authority>, <xref:System.UriPartial.Path>, or <xref:System.UriPartial.Query>.
 
 |URI|Scheme|Authority|Path|Query|
 |---------|------------|---------------|----------|-----------|
 |`http://www.contoso.com/index.htm?date=today`|`http://`|`http://www.contoso.com`|`http://www.contoso.com/index.htm`|`http://www.contoso.com/index.htm?date=today`|
 |`http://www.contoso.com/index.htm#main`|`http://`|`http://www.contoso.com`|`http://www.contoso.com/index.htm`|`http://www.contoso.com/index.htm`|
-|`mailto:user@contoso.com?subject=uri`|`mailto:`|`mailto:user@contoso.com`|`mailto:user@contoso.com?subject=uri`|`<none>`|
+|`mailto:user@contoso.com?subject=uri`|`mailto:`|`<none>`|`mailto:user@contoso.com`|`mailto:user@contoso.com?subject=uri`|
```
Comment (MihaZupan) [inline xml/System/Uri.xml:2413]: Your table looks correct, but this case seems to be highlighting a bug in Uri

These outputs don't make sense
```c#
var uri = new Uri("mailto:user@contoso.com?subject=uri");
uri.GetLeftPart(UriPartial.Authority).Dump(); // Empty -- should be mailto:user@contoso.com
uri.GetLeftPart(UriPartial.Path).Dump();      // mailto:user@contoso.com
```

Filed https://github.com/dotnet/runtime/issues/89262

File: xml/System/Uri.xml [hunk 2]
```diff
@@ -2394,27 +2394,25 @@ If you used an escaped string to construct this instance (for example, `"http://
...
- <xref:System.Uri.GetLeftPart%2A> includes delimiters in the following cases:
+The <xref:System.Uri.GetLeftPart%2A> method returns a string containing the leftmost portion of the URI string, ending with the portion specified by `part`.
 
--   <xref:System.UriPartial.Scheme> includes the scheme delimiter.
+<xref:System.Uri.GetLeftPart%2A> includes delimiters in the following cases:
 
--   <xref:System.UriPartial.Authority> does not include the path delimiter.
+- <xref:System.UriPartial.Scheme> includes the scheme delimiter.
+- <xref:System.UriPartial.Authority> does not include the path delimiter.
+- <xref:System.UriPartial.Path> includes any delimiters in the original URI up to the query or fragment delimiter.
```
Comment (MihaZupan) [inline xml/System/Uri.xml:2404]:
```suggestion
- <xref:System.UriPartial.Path> includes the path delimiter and any delimiters in the original URI up to the query or fragment delimiter.
```

E.g. For `http://foo`, `UriPartial.Path` will insert the path delimiter `http://foo/`

**Review by MihaZupan — APPROVED** [review]
Thanks

**Pull request merged**